app/views/account/workspaces/_show_user.html.slim

```diff
@@ -0,0 +1,9 @@
+- users.each do |member|
+  div data-attribute = "#{member.id}"
+    = fa_icon "user-o"
```

Comment:
fa_icon 'user-o', text: member.full_name you can do it like this and remove not needed span